shared/src/packages/orders/libs/types/order-response-dto.type.ts

```diff
@@ -0,0 +1,22 @@
+import { type DriverEntity } from '~/packages/drivers/drivers.js';
+import { type ShiftEntity } from '~/packages/shifts/shifts.js';
+import { type TruckEntity } from '~/packages/trucks/trucks.js';
+import { type UserEntityT } from '~/packages/users/users.js';
+
+import { type OrderEntity } from './order-entity.type.js';
+
+type OrderResponseDto = Omit<OrderEntity, 'shift' | 'driver' | 'truck'> & {
+  shift: {
+    id: ShiftEntity['id'];
```

Comment:
1. There's no need to omit 'shift', because if you combine types via '&', then their properties are merged.
2. If we don't omit 'shift', then we should remove  "id: ShiftEntity['id'];"